critiquebrainz/chefignore

```diff
@@ -0,0 +1,102 @@
+# Put files/directories that should be ignored in this file when uploading
```

Comment:
I wonder if there's a way to have all the cookbooks share this, since it seems generally useful. Maybe symlinks.